src/peergos/server/net/GatewayHandler.java

```diff
@@ -16,15 +16,46 @@
 public class GatewayHandler implements HttpHandler {
 	private static final Logger LOG = Logging.LOG();
 	private static final boolean LOGGING = true;
+	private static final int MAX_ASSET_SIZE_CACHE = 200*1024;
 
     private final String domainSuffix;
     private final NetworkAccess network;
     private final Crypto crypto;
+    private final LRUCache<String, FileWrapper> webRootCache;
+    private final LRUCache<String, Asset> assetCache;
 
     public GatewayHandler(String domainSuffix, Crypto crypto, NetworkAccess network) {
         this.domainSuffix = domainSuffix;
         this.crypto = crypto;
         this.network = network;
+        this.webRootCache = new LRUCache<>(1000);
```

Comment:
paramaterise size?

Reply:
It's not random, it's chosen based on the maximum amount of ram I want the cache to use.

Reply:
And the number of users we have